great_expectations/core/usage_statistics/usage_statistics.py

```diff
@@ -187,7 +189,7 @@ def build_envelope(self, message):
             + "Z"
         )
 
-        event_duration_property_name: str = f'{message["event_name"]}.duration'.replace(
+        event_duration_property_name: str = f'{message["event"]}.duration'.replace(
```

Comment:
@anthonyburdi I am sure you are right -- it just seems a little surprising that it has worked with our "standard" tests before (with the key being "event_name") -- are you sure about this?

Reply:
I thought so too, but this was a recent addition in https://github.com/great-expectations/great_expectations/pull/3729 - we have used `event_name` as a parameter in many places but the key has always been `event`.

Reply:
@anthonyburdi So maybe that was the issue, meaning that while the schema and message tests worked, the events were never actually emitted?..  Is that what is happening here?